sass/layouts/services.scss

```diff
@@ -0,0 +1,45 @@
+.services {
+  background-color: $color-services-background;
+  border-bottom: 6px solid $color-services-shadow;
+  position: relative;
+  top: 66px;
+  @include media-768 {
+    top: 47px;  
+    border-bottom: none;
+  }
+}
+
+.services-header {
+  padding: 59px 44px 60px 40px;
+  box-sizing: padding-box;
+  @include media-1020 {
+    padding: 39px 41px 28px 42px;
+  }
+  @include media-768 {
+    padding: 25px 30px;
+  }
+  @include media-375 {
+    padding: 25px 15px;
+  }
+}
+
+.services-header p {
+  margin-top: 18px;
+}
+
+.services-list {
+  padding: 0 40px 60px;
+  @include media-768 {
+    padding: 0 33px 15px;
+  }
+  @include media-375 {
+    padding: 0 15px 33px;  }
```

Comment:
Code style. Why is the closing bracket on the same line?

Reply:
DONE!